docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3359,6 +3363,21 @@ for the table entry "executes" the direct resource appropriately.
         * if **set**: The response includes the counter entry's value read from
           the target.
 
+* `meter_counter_data` field
+    * `WriteRequest (INSERT)`
+        * if **unset**: The initial value for all 3 counter entries is the default
+          (0).
+        * if **set**: The initial value for all 3 counter entries is the default (0).
+        Sub-field values, if any set, are ignored. 
+    * `WriteRequest (MODIFY)`
+        * if **unset**: All the 3 counter entry's value is not changed.
+        * if **set**: All the 3 counter entry's value is reset to 0. 
+    * `ReadRequest`
+        * if **unset**: The response does not include the counter entry's value
+          (`meter_counter_data` is unset in the response).
+        * if **set**: The response includes all the 3 counter entry's value read from
+          the target.
```

Comment:
same comment as above, grammatically incorrect. Maybe: "The response includes all 3 counter values read from the target"